### DISCUSSION TITLE
HTTPS

### DISCUSSION THREAD

**Discussion by tanchiawei** [hktalent/scan4all#108] [Q&A]
目标服务器使用 https 协议而不是 http 协议。 尝试使用 scan4all 时，无法连接到 Web 服务器

![image](https://user-images.githubusercontent.com/72211145/189501208-5d64c677-417e-4bbe-9e8b-2fe446990de0.png)

**Comment by hktalent** [comment]
./scan4all -host https://51pwn.com -v
<img width="993" alt="image" src="https://user-images.githubusercontent.com/18223385/189507886-fc8af78d-23e8-41d1-a594-f7c914db30bf.png">
